listenbrainz/follow_server/dispatcher.py

```diff
@@ -3,24 +3,41 @@
 import pika
 import time
 import threading
+import socketio
 
 from flask import current_app
-
+from listenbrainz.webserver.views.api_tools import LISTEN_TYPE_PLAYING_NOW, LISTEN_TYPE_IMPORT
 
 class FollowDispatcher(threading.Thread):
 
     def __init__(self, app):
         threading.Thread.__init__(self)
         self.app = app
 
+    def send_listens(self, listens, listen_type):
+        sio = socketio.Client()
```

Comment:
Not sure how to best do this without creating a new connection to the server everytime. Do this in another thread?